Makefile

```diff
@@ -328,17 +328,13 @@ create-cluster: $(CLUSTERCTL) ## Create a development Kubernetes cluster on AWS
 	# Create control plane machine.
 	kubectl \
 		create -f examples/_out/controlplane.yaml
-	# Get KubeConfig using clusterctl.
-	$(CLUSTERCTL) \
-		alpha phases get-kubeconfig -v=3 \
-		--kubeconfig=$${HOME}/.kube/config \
-		--namespace=default \
-		--cluster-name=$(CLUSTER_NAME)
+	# Wait for the kubeconfig to become available.
+	timeout 300 bash -c "while ! kubectl get secrets | grep test1-kubeconfig; do sleep 1; done"
+	# Get kubeconfig and store it locally.
+	kubectl get secrets test1-kubeconfig -o json | jq -r .data.value | base64 -D > ./kubeconfig
 	# Apply addons on the target cluster, waiting for the control-plane to become available.
-	$(CLUSTERCTL) \
-		alpha phases apply-addons -v=3 \
-		--kubeconfig=./kubeconfig \
-		-a examples/addons.yaml
+	timeout 300 bash -c "while ! kubectl --kubeconfig=./kubeconfig get nodes | grep master; do sleep 1; done"
```

Comment:
Can we do a query using labels or something rather than grepping for the master node role?